README.rst

```diff
@@ -0,0 +1,125 @@
+Free Text Response XBlock
+==================
+
+XBlock to capture a free-text response.
+
+|badge-travis|
+|badge-coveralls|
+
+This package provides an XBlock for use with the EdX Platform and makes
+it possible for instructors to create questions that expect a
+free-text response.
+
+|image-lms-view-normal|
+
+
+Installation
+------------
+
+
+System Administrator
+~~~~~~~~~~~~~~~~~~~~
+
+To install the XBlock on your platform,
+add the following to your `requirements.txt` file:
+
+    xblock-free-text-response
+
+You'll also need to add this to your `INSTALLED_APPS`:
+
+    freetextresponse
```

Comment:
Not really. Why would we need this anyway?

Reply:
XBlock uses templates in a self-contained way, so this is not needed.

Unless there's migration, or platform-wide templates (plus other minor uses) `INSTALLED_APPS` is not needed